src/game/shared/swarm/asw_gamerules.cpp

```diff
@@ -180,10 +180,18 @@ extern ConVar old_radius_damage;
 	ConVar rd_sentry_block_aliens( "rd_sentry_block_aliens", "1", FCVAR_CHEAT, "If 0 sentries don't collide with aliens" );
 	ConVar rd_auto_fast_restart( "rd_auto_fast_restart", "0", FCVAR_NONE, "Set to 1 to restart mission on fail automatically" );
 	ConVar rd_adjust_mod_dont_load_vertices("rd_adjust_mod_dont_load_vertices", "1", FCVAR_NONE, "Automatically disables loading of vertex data.", true, 0, true, 1);
-	ConVar rd_high_resolution_timer_ms ( "rd_dedicated_high_resolution_timer_ms", "0.1", FCVAR_NONE, "Acquire a high resolution timer with specified resolution." );
+	ConVar rd_high_resolution_timer_ms( "rd_dedicated_high_resolution_timer_ms", "0.01", FCVAR_NONE, "Acquire timer with specified resolution in ms" );
```

Comment:
Is there a reason why ConVar in code rd_high_resolution_timer_ms differs in name with the console version rd_dedicated_high_resolution_timer_ms?

Reply:
No reason. Just pushed a commit to sync the names.